File: trunk/src/examples/python/example2.py
import pyscovil, time, random

curent_options = pyscovil.options(640, 480, 32, False, True, "test", "Test")
scene_object = pyscovil.scene()

dots = []
for i in xrange(1000):
	cords =  [random.uniform(1, 4)-2, random.uniform(1, 4)-2, -1]
	dots.append(pyscovil.dot(scene_object, cords, int(random.uniform(2, 6))))


time.sleep(2)

for i in xrange(1000):
	dots[i].set_cord([dots[i].cord[0]-random.random()+0.5, dots[i].cord[1]-random.random()+0.5, dots[i].cord[2]])

time.sleep(5)

for i in xrange(500):
	scene_object.delete_lower(dots[0], True)
	dots.pop(0)
time.sleep(5)

File: trunk/src/examples/ruby/example2.rb
require 'rubyscovil'
require 'ruinterscovil'
curent_options = Options.new.init(640,480,32,1,0,"Test","Test")
scene_object = Scene.new.init()
#camera1 = Camera.new.init()
#Ruinterscovil.create_change_camera(nil);
#Ruinterscovil.create_change_camera()
sleep (1)

dot1=Dot.new
#dot1.dot_init()

sleep(1)
dot1.dot_size(50,true)
sleep(1)

dot1.dot_cord([0,-0.7,-1],true)
sleep(1)

dot1.dot_color([0.0, 0.0, 1.0, 1.0],true)
sleep(1)

dot2=Dot.new
#dot2.dot_init()

sleep(1)
dot2.dot_cord([0,0.7,-1],true)
sleep(1)
dot2.dot_color([1.0, 0.0, 0.0, 1.0],true)
sleep(1)
#dot2.dot_upper(dot1, true)
#sleep(1)

#dot2.dot_mat_cord([0, -0.9, -1],true)

#dot1.dot_size(100, true)
#quad1=Quad.new

#quad1.quad_init()
sleep(1)
#tri1.triangle_delete(tri1,true)
#dot2=Dot.new.init(dot1,[0,0,-1],100)

#dot1.delete(dot2, true)

camera1 = Camera.new()
camera1.look_at()
#camera1.set_current(true)


sleep(3)




